tests/traces/README

```diff
@@ -5,3 +5,4 @@ We collect them here for convenience only.
 
 PE
 - [ftp-pe.pcap](https://github.com/zeek/zeek/blob/master/testing/btest/Traces/pe/pe.trace) (pruned to a single file transfer)
+- [http-sample-dll.pcap]() Manually created by @fox-ds. A MinGW-compiled DLL on Windows 10 x64 that imports several functions and exports a single function, hosted on a Python HTTP server and downloaded via HTTP. Added under the MIT license.
```

Comment:
I think it would simplify things a lot if there was just a single license for this package.

AFAICT the [Zeek license used in this project](https://github.com/zeek/spicy-pe/blob/main/LICENSE) (basically three-clause BSD) is compatible with MIT.  Could you consider changing the license of the PCAP file to be in line with the rest of the package? Alternatively, having the PCAP dual-licensed under both the license of this project and MIT would also work.

Reply:
Yes, after reviewing I think changing the license to three-clause BSD is pretty similar to MIT, so I'll just remove the last sentence so that it just implicitly uses the project's license.